glotaran/project/optimization_history.py

```diff
@@ -0,0 +1,131 @@
+"""Module containing the ``OptimizationHistory`` class."""
```

Comment:
Move this module to the optimization package.

Reply:
Done in [1545bf0](https://github.com/glotaran/pyglotaran/pull/1134/commits/1545bf03a240ea72f39b05d8a2474080e01359a3)